12-watchtower-API.md

```diff
@@ -127,122 +129,167 @@ This message contains all the information regarding the appointment that the cli
+	* `op_customer_signature` is missing.
+	* `op_customer_public_key` is missing.
+	* `op_customer_signature` cannot be verified using `op_customer_public_key`.
+
+* if `qos` was NOT agreed on `wt_init`:
+	* The receiving node SHOULD also reject the appointment if:
+	* if `op_customer_signature_algorithm` is present.
+	* if `op_customer_signature` is present.
+	* if `op_customer_public_key` is present.
+
+The receiving node MAY accept the appointment otherwise.
+
+#### Range of values 
+
+- `start_block`: Absolute Block number 
+- `end_block`: Absolute Block number
+- `dispute_delta`: Relative Block number
+- `transaction_size`: Measured in Bytes
+- `transaction_fee`: Measured in sats
+- `cipher`: AESGCM256, CHACHA20
+- `customer_signature_algorithm`: ECDSA, SCHNORR
 
-	
 #### Rationale
+
+We can group the data fields into logical groups. 
+
+* **Appointment information**: The appointment time is defined using the `start_block` and `end_block`. WatchTower will delete the job when the appointment has expired. We recommend only using block numbers as that is the natural clock for Bitcoin. 
+* **Explicit acknowledgement of transaction details**: The `dispute_delta`, `transaction_size`, and `transaction_fee` let the WatchTower confirm the transaction is "reasonable" and it can be accepted into the blockchain (especially if there is congestion in the future). 
+* **Encrypted transaction**: The `cipher`,`encrypted_blob` states how the WatchTower can later find the dispute transaction and decrypt the justice transaction. 
+* **Customer signature** The `customer_public_key` and `customer_signature` provides an explicit message about the job from the customer.  The `customer_public_key` can also be used for refunds if applicable.
+
 The transaction `locator` can be deterministically computed by both the client and the server. Locators of wrong size are therefore invalid.
 
 `start_block` and `end_block` too close to the current chain tip may result in the tower missing the trigger and therefore should be avoided.
 
-Too far away is a subjective concept. Towers accepting jobs that will start in the far future or that may last a really long time risk having to store data for long periods of time.
+Too far away is a subjective concept. Towers accepting jobs that start in the far future or that may last a really long time risk having to store data for long periods of time and should, therefore, by avoided.
 
-The concept of too small for `dispute_delta` is also subjective. The `dispute_delta` defines how many blocks the server will have to respond with the `justice_transaction`. The smallest the value, the more the server risks to fail the appointment.
+The concept of too small for `dispute_delta` is also subjective. The `dispute_delta` defines how many blocks the server will have to respond with the `justice_transaction` after a breach is seen. The smallest the value, the more the server risks to fail the appointment.
```

Comment:
do we need 'also'?

Reply:
We come from a paragraph were we talk about something subjective. We are talking about another subjective thing in this paragraph. 'also' seems reasonable in this situation.

Reply:
their bullet points right

Reply:
No, they are not here. They are different paragraphs in the rationale